tests/e2e/scripts/execute.sh

```diff
@@ -22,15 +22,30 @@ cd $workdir
 curpath=$PWD
 echo $PWD
 
-which ginkgo &> /dev/null || (
-    go install github.com/onsi/ginkgo/ginkgo@latest
-    sudo cp $GOPATH/bin/ginkgo /usr/local/bin/
-)
+check_ginkgo() {
+    # check if installed
```

Comment:
Yes, I check the version in the next step. 

It follows steps like:
1. check if ginkgo cmd exists, otherwise install ginkgo v2
2. check the ginkgo version of existing ginkgo cmd, if it's not ginkgo v2, reinstall it.

Reply:
I just meant that your comment there could be more accurate. :)